src/main/java/de/zalando/zmon/dataservice/data/CheckData.java

```diff
@@ -18,4 +18,5 @@
     public JsonNode check_result;
     public boolean exception;
     public Map<String, AlertData> alerts = new HashMap<>(0);
+    public boolean is_sampled = true;
```

Comment:
Java's convention is to have names of fields in methods in camelCase

Reply:
If this class represents deserialized JSON object, it makes sense to use `@JsonProperty` annotation to define matching between objects' fields

Reply:
Fixed.